iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ConnectDisconnectUtil.java

```diff
@@ -28,6 +28,15 @@ public final class ConnectDisconnectUtil {
     private ConnectDisconnectUtil() {
     }
 
+    /**
+     * Connect the specified terminals of the identifiable. It will connect all the specified terminals or none if at
```

Comment:
terminals of the identifiable does not sound good. Terminals are related to connectable, exception for HVDC line and tie line only.